src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/PremisesController.kt

```diff
@@ -261,24 +262,12 @@ class PremisesController(
     }
 
     if (body.departureDate.isBefore(body.arrivalDate)) {
-      validationErrors["$.departureDate"] = "departureBeforeArrival"
+      validationErrors["$.departureDate"] = "beforeBookingArrivalDate"
```

Comment:
I believe there's a map in the frontends for these keys - does this need a corresponding change in `hmpps-approved-premises-ui`?

Reply:
This check was introduced specifically for Temporary Accommodation a little while back - it's just being changed to be consistent with the key used when creating an arrival. I just did a check and `"departureBeforeArrival"` doesn't show up anywhere in `hmpps-approved-premises-ui` so it shouldn't cause any issues there.